<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>forecast_base</class>
 <widget class="QMainWindow" name="forecast_base">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>625</width>
    <height>545</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>625</width>
    <height>545</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>forecast_base</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>img/app_icon.png</normaloff>img/app_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #31CCFE	;
QScrollArea{
background-color :rgb(255, 69, 0);
border: 2px solid #FEFEFE;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QScrollArea" name="scrollArea_OWM">
      <property name="accessibleDescription">
       <string/>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color :rgb(255, 69, 0);
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_OWM">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>605</width>
         <height>259</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="bottomMargin">
         <number>9</number>
        </property>
        <item>
         <widget class="QFrame" name="frame_OWM">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color :rgb(255, 69, 0);
color: white;
border-radius: 15px;
border: 6px solid #FEFEFE;
background-color :rgb(255, 69, 0);
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLabel" name="OWM_data_temp">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>120</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>35</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color :rgb(255, 69, 0);
color: white;
border: none;</string>
             </property>
             <property name="text">
              <string>t</string>
             </property>
             <property name="textFormat">
              <enum>Qt::MarkdownText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_OWM">
             <item>
              <widget class="QLabel" name="OWM_data_weather">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="tabletTracking">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
               </property>
               <property name="text">
                <string>weather </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="OWM_data_humidity">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <stylestrategy>PreferDefault</stylestrategy>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
               </property>
               <property name="text">
                <string>h</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="OWM_data_date">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
             </property>
             <property name="text">
              <string>dd:mm:yyyy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea_WeatherBit">
      <property name="styleSheet">
       <string notr="true">background-color :rgb(255, 69, 0);
</string>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarAsNeeded</enum>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents_WeatherBit">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>605</width>
         <height>258</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QFrame" name="frame_WeatherBit">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>240</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color :rgb(255, 69, 0);
color: white;
border-radius: 15px;
border: 6px solid #FEFEFE;
background-color :rgb(255, 69, 0);
</string>
          </property>
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="WeatherBit_data_temp">
             <property name="minimumSize">
              <size>
               <width>100</width>
               <height>120</height>
              </size>
             </property>
             <property name="font">
              <font>
               <pointsize>35</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color :rgb(255, 69, 0);
color: white;
border: none;</string>
             </property>
             <property name="text">
              <string>t</string>
             </property>
             <property name="textFormat">
              <enum>Qt::MarkdownText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
             <property name="wordWrap">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_WeatherBit">
             <item>
              <widget class="QLabel" name="WeatherBit_data_weather">
               <property name="minimumSize">
                <size>
                 <width>80</width>
                 <height>35</height>
                </size>
               </property>
               <property name="font">
                <font>
                 <pointsize>11</pointsize>
                </font>
               </property>
               <property name="tabletTracking">
                <bool>false</bool>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
               </property>
               <property name="text">
                <string>weather </string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="WeatherBit_data_humidity">
               <property name="font">
                <font>
                 <pointsize>13</pointsize>
                 <weight>75</weight>
                 <italic>false</italic>
                 <bold>true</bold>
                 <underline>false</underline>
                 <strikeout>false</strikeout>
                 <stylestrategy>PreferDefault</stylestrategy>
                 <kerning>true</kerning>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
               </property>
               <property name="text">
                <string>h</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="WeatherBit_data_date">
             <property name="minimumSize">
              <size>
               <width>110</width>
               <height>20</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color :rgb(255, 69, 0);
color:white;
border: none;</string>
             </property>
             <property name="text">
              <string>dd:mm:yyyy</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
